src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Resources.resx

```diff
@@ -118,73 +118,73 @@
-    <value>Provided assembly search directory '{0}' does not exist.</value>
+    <value>Provided search directory '{0}' does not exist.</value>
   </data>
   <data name="CannotAddAbstractMember" xml:space="preserve">
-    <value>Cannot add abstract member '{0}' to {1} because it does not exist on {2}</value>
+    <value>Cannot add abstract member '{0}' to {1}.</value>
   </data>
   <data name="AssemblyCultureDoesNotMatch" xml:space="preserve">
-    <value>{2} assembly culture '{0}' does not match with {3} assembly culture '{1}'.</value>
+    <value>'{0}' assembly culture does not match '{1}'.</value>
   </data>
   <data name="AssemblyVersionDoesNotMatch" xml:space="preserve">
-    <value>{3} assembly version '{1}' should be equal to {2} version '{0}'.</value>
+    <value>'{1}' assembly version should be '{0}'.</value>
   </data>
   <data name="AssemblyVersionIsNotCompatible" xml:space="preserve">
-    <value>{2} assembly version '{0}' should be equal to or higher than {3} version '{1}'.</value>
+    <value>'{0}' assembly version should be equal to or higher than '{1}'.</value>
   </data>
   <data name="ElementShouldNotBeNullAtIndex" xml:space="preserve">
-    <value>Element should not be null. Found null at index: {0}.</value>
+    <value>Element should not be null at index: {0}.</value>
   </data>
   <data name="IndexShouldBeWithinSetSizeRange" xml:space="preserve">
-    <value>The index should be within the right set size range.</value>
+    <value>Index should be within the right set size range.</value>
   </data>
   <data name="MatchingAssemblyNotFound" xml:space="preserve">
-    <value>Could not find matching assembly: '{0}' in any of the search directories.</value>
+    <value>Could not find matching assembly: '{0}'.</value>
   </data>
   <data name="MemberExistsOnLeft" xml:space="preserve">
-    <value>Member '{0}' exists on {1} but not on {2}</value>
+    <value>Member '{0}' exists on {1} but not on {2}.</value>
   </data>
   <data name="MemberExistsOnRight" xml:space="preserve">
-    <value>Member '{0}' exists on {2} but not on {1}</value>
+    <value>Member '{0}' exists on {2} but not on {1}.</value>
   </data>
   <data name="RightNamesAtLeastOne" xml:space="preserve">
-    <value>Should at least contain one right name.</value>
+    <value>Should contain at least one right name.</value>
   </data>
   <data name="ShouldBeGreaterThanZero" xml:space="preserve">
     <value>Value should be greater than 0.</value>
   </data>
   <data name="TypeMissingOnSide" xml:space="preserve">
-    <value>Type '{0}' exists on {1} but not on {2}</value>
+    <value>Type '{0}' exists on {1} but not on {2}.</value>
   </data>
   <data name="VisitorRightCountShouldMatchMappersSetSize" xml:space="preserve">
-    <value>The provided right count when creating the visitor should match the right set size specified for the mappers.</value>
+    <value>Provided right count should match right set size.</value>
```

Comment:
This doesn't read right. Maybe "Right count that was provided should .." or "Right count should.." .. can't suggest as I don't know what it means.

Reply:
I looked for the implementation of this rule in the project and I couldn't find one. So I also don't know what it means. I guess its better leave this one as it is

Reply:
Nice question...

Reply:
> The `throw` statement that used this string seems to have been deleted in https://github.com/dotnet/sdk/pull/27203/files#diff-be4b35510e0709369d0d66665b37d00e8090ce0da494e411bdeacaab8e5c3609L131. Just delete the string resource, then?

I took it off :)